mhs/common/mhs_common/errors/ebxml_handler.py

```diff
@@ -78,12 +79,16 @@ def handle_ebxml_error(code: int, headers: Dict, body: AnyStr) -> Tuple[int, Opt
 
     ebxml_error_envelope: EbxmlErrorEnvelope = EbxmlErrorEnvelope.from_string(body)
 
-    errors_text = ''
+    error_data_response = {'error_message': 'Error(s) received from Spine. Contact system administrator.',
+                           'error_code': '0005',
```

Comment:
I assume the error_code is the "magic number" people used to add in logs, and process_key is the logger name?
Do we need this error_code and process_key here anyway? I would like to remove this kind of artificial identifiers

Reply:
yes, I was basing on log msg. I will remove this as it's not needed anymore.